packages/common/src/client/clamav-rest-client.js

```diff
@@ -0,0 +1,53 @@
+const axios = require('axios').default;
+const FormData = require('form-data');
+// const { Readable } = require('stream');
+const fs = require('fs');
+
+const trailingSlashRegex = /\/$/;
+
+class ClamAVClient {
+	/**
+	 * @param {string} host - e.g. https://example.com
+	 * @param {number} [maxBodyLength] - maxBodySize in bytes used by axios
+	 */
+	constructor(host, maxBodyLength) {
+		if (!host) {
+			throw new Error('host is required');
+		}
+
+		/** @type {string} */
+		this.host = host.replace(trailingSlashRegex, '');
+		this.maxBodyLength = maxBodyLength ? maxBodyLength * 1.1 : maxBodyLength; // add headroom
+	}
+
+	/**
+	 * sends file to clamav rest server
+	 * @param {import('express-fileupload').UploadedFile} fileInformation
+	 * @param {string} fileName
+	 * @returns {Promise<boolean>}
+	 */
+	async scan(fileInformation, fileName) {
+		if (typeof fileInformation?.tempFilePath !== 'undefined') {
+			const readableStream = fs.createReadStream(fileInformation?.tempFilePath);
```

Comment:
have modified this from believe this should be streams all the way down now?

    const fileBuffer = fs.readFileSync(fileInformation?.tempFilePath);
    const readableStream = Readable.from(fileBuffer.toString());